tests/schemas/dump02_default.esdl

```diff
@@ -69,3 +69,10 @@ type Tree {
     link children := .<parent[IS Tree];
     property child_vals := .children.val;
 }
+
```

Comment:
Is adding tests/dumps/dump02/1_0a8.dump intentional? We don't have a8, so it's confusing.

Reply:
I can't generate it from a7 due to migration changes, so a8 is the best approximation of state between a7 and b1